src/merge/mod.rs

```diff
@@ -1,12 +1,15 @@
+extern crate itertools;
+
 mod chunk_iterator;
 mod chunk;
 mod output;
 
 use std::fmt::Debug;
 
-use diff;
 
+use diff;
```

Comment:
The `use`s are grouped: imports from `std` first, imports from external crates and imports from other modules. Please retain this grouping (ie. revert this change)

Reply:
done